spec/visual_spec.rb

```diff
@@ -81,7 +81,6 @@ def create_element(name, text)
       it 'excludes links that do not resolve' do
         links = JSON.parse(subject.send(:get_links, check_links))
         expect(links.has_key?('http://google.com')).to be true
-        expect(links.has_key?('http://libweb5.princeton.edu/visual_materials/ga/bad link.jpg')).to be false
```

Comment:
This test doesn't seem related to the other stuff in the PR. I'm not sure why it's being deleted?

Reply:
The test refers to a server that doesn't exist anymore. I confirmed with @kevinreiss .

Reply:
> See my one question in line.
> 
> Do we need to merge this before we can try indexing them into the alma qa site?
> 
> Do we not need a branch or anything here because we use release numbers?

If we merge it we create a tag. The gem is referenced in the orangelight gemfile with a tag.
I indexed yesterday 3 example thesis records for @kevinreiss and @carolyncole to test them. They both confirmed in slack that they look ok with the requests and they can get to the aeon form.

Reply:
great, thanks!